includes/definitions/hwg-wld.yaml

```diff
@@ -0,0 +1,42 @@
+os: hwg-wld
+text: 'HWg WLD'
+type: environment
+icon: hwg
+mib_dir: hwg
+over:
+    - { graph: device_state, text: States }
+nobulk: true
+poller_modules:
```

Comment:
Please don't do this in OS.  It blocks users global settings.

Reply:
I kept whatever was existing in the original OS, but as I own one WLD device, I'll check if it is really necessary. Basically, the complete snmpwalk of the device is 10 lines long (dumb device, really, does only what it is meant to do), meaning that any discovery or poller will essentially timeout :) I think this was the reason the original device included a lot of disabled modules.

Reply:
I cleaned hwg-wld.yaml for module restrictions. It works, but it has an impact on the poller time (from 600 ms to 1.8 s). It is definitely a waste of time because clearly the device has only 10 lines in the snmpwalk. Won't change anytime soon. 
Let me know what you prefer. I would personnaly keep the restrictions for now, and if, one day, they create a new model, we'll always be accepting a new PR to remove the restrictions.

Reply:
Which modules?  Poller modules that aren't polling data should not add to the polling time.  I have tried to make that the case for most of them.